### DISCUSSION TITLE
any tailwind experts here? simple question - what in the web is this, or what it do, to anyone that knows abosutley nothing about css ?

### DISCUSSION THREAD

**Discussion by needhelpfulanswers-spectrumchat** [tailwindlabs/tailwindcss#2964] [General]
anyone here know about this tailwind thing? 

what in the web is this to anyone that knows abosutley nothing about css ?

what are at least 3 non-code examples / analogies of why this tailwind  thing is better ?

like is it a new way to do css ? or is it something that replaces css? or is it basically css?
is it anothre way to do css ? like there's different ways to speak englisH ?

what are at least 3 non-code examples / analogies of why this tailwind  thing is better ? that  anyone that knows abosutley nothing about css would understand??

no rush, within next 3 or 10 years is fine

if anyone doesnt know how to show/explain  at least 3 non-code examples / analogies of why this tailwind  thing is better to anyone that knows abosutley nothing about css would understand....

then it's very likely they dont have good communciaiton skills and that can be made lots better, and they very likely dont understand this tailwind thing

for anyone to really understand anything, they would need to have  good communciaiton skills 

if anyone doesnt know how to show/explain  at least 3 non-code examples / analogies of why this tailwind  thing is better to anyone that knows abosutley nothing about css would understand - then their comucniation skills could be made lots bettter

---

a person cannot prove or verify they actually understand this tailwind thing if they arent even able to communicate it to anyone that knows abosutely nothing about css

what are at least 3 non-coding examples / analogies of why this tailwind  thing is better so that anyone that knows abosutley nothing about css would understand?

should anyone learn css this way via tailwind? is the basically the same way css is normally learnt? or is it a different way? is tailwind a different language, not css?

----

no rush, anytime, in 10 years is fine

show in baby steps like we're 4 what this tailwind thing do to the 99% of the world population that has 0 no idea what css or technology is

if a person doesnt have the basic skills to do that simple thing, then there's no way for them understand if tailwind is good or bad overall

and there's no way tailwind could be good. if sometihng was actually truely best/good, then a person would have a high level of skills in all areas - cos to be able to really understand best/good things, a person would need  a high level of skills in all areas to even have the capacity to be able to use best/good things well

to be able to use soemthing well, they have to be able to understand it (X) something, anything

1. so if a person actually did understand a topic or anything
2. then they're be able to use or be able to show that thing very very well
3. so that anyone else are able to easily understand what you're saying about how this tailwind is better/best

by way of   at least 3 non-code examples / analogies of why this tailwind  thing is better to anyone that knows abosutley nothing about css or tech or w/e this is,

or does

---

no rush, 10 or 100+ or 100000+ years to up your skills - and to make simple progress - on basic things like being able to communicate well so that anyone that knows aboustely nothing about what a broswer is or anything can understand what you are saying about this tailwind thing  

if you cannot show to anyone else that you know god, then you dont know god
if you cannot show to anyone else that you know science, then you dont know science
if you cannot show to anyone else that you know css or code, then you dont know code
if you cannot show to anyone else that you know good art or good ui/ux/design, then you have no idea what good ui/ux/design is

if you cant even show basic things to anyone else that has no idea what you're talking about, especially a 4y old

**Comment by simonswiss** [comment]
Hi, 

I read all of this, but I need at least three non-code examples / analogies of what exactly you're asking 😅

**Reply by MartinHughes-BPC** [reply]
I was planning on getting back to him in 10,000 years 🥇
